libr/anal/hint.c

```diff
@@ -55,6 +55,9 @@ static void setHint(RAnal *a, const char *type, ut64 addr, const char *s, ut64 p
 	}
 }
 
+R_API void r_anal_hint_set_offset(RAnal *a, ut64 addr, const char* typeoff) {
+	setHint (a, "Offset:", addr, r_str_trim_const (typeoff), 0);
+}
 R_API void r_anal_hint_set_jump(RAnal *a, ut64 addr, ut64 ptr) {
```

Comment:
Missing newline

Reply:
The all functions below missed newline too, so I just kept the same style